hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -437,17 +448,22 @@ private String addToBatch(Queue<Entry> buffer, BatchOperation batchOperation) {
    */
   private List<Queue<Entry>> splitReadyBufferAtCreateSnapshot() {
     final List<Queue<Entry>> response = new ArrayList<>();
-
+    final Set<OzoneManagerProtocolProtos.Type> standaloneBatchCmdTypes = ImmutableSet.of(
+        OzoneManagerProtocolProtos.Type.SnapshotPurge, OzoneManagerProtocolProtos.Type.CreateSnapshot);
+    final List<Function<OMResponse, Boolean>> standaloneBatchConditions =
+        ImmutableList.of(OMResponse::hasCreateSnapshotResponse,
```

Comment:
Why do we need to check `OMResponse::hasCreateSnapshotResponse`? Can't we just rely on `CmdType`?

Reply:
this was the previous logic. I can remove it.

Reply:
done